src/apps/btc/btc_sign.c

```diff
@@ -263,6 +307,22 @@ static app_btc_sign_error_t _sign_input_pass2(
                 return _error(APP_BTC_SIGN_ERR_INVALID_INPUT);
             }
             break;
+        case BTCScriptConfig_multisig_tag: {
+            uint8_t sighash_script_tmp[MAX_PK_SCRIPT_SIZE - MAX_VARINT_SIZE] = {0};
```

Comment:
Can you explain why this is MAX_PK_SCRIPT_SIZE *minus* MAX_VARINT_SIZE?

Reply:
I can't, because it's wrong :P nice catch! before it was `sizeof(sighash_script) - MAX_VARINT_SIZE`, but that's just `MAX_PK_SCRIPT_SIZE`.

Fixed.